addons/actions/src/util.js

```diff
@@ -113,5 +123,7 @@ export function retrocycle($) {
     }
   })($);
 
+  muteProperty(CYCLIC_KEY, $);
```

Comment:
@rhalff this breaks when `$` is a primitive value (e.g., a string). The failure is silent because of this catch block: https://github.com/storybooks/storybook/blob/master/lib/channel-postmessage/src/index.js#L68